fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -381,7 +381,7 @@ showDataStmt
 // ------------------------------------------- Table Statement ---------------------------------------------------------
 
 createTableStatement
-    : CREATE EXTERNAL? TABLE (IF NOT EXISTS)? qualifiedName
+    : CREATE EXTERNAL? TEMPORARY? TABLE (IF NOT EXISTS)? qualifiedName
```

Comment:
it should be `CREATE { TEMPORARY | TEMP } EXTERNAL TABLE`

Reply:
1. that will be handled in spark connector, e.g. creating table with uuid suffix
2. Manual deletion seems acceptable, I prefer not introducing something new in FE metadata right now, which could be handled later.

Reply:
> it should be `CREATE { TEMPORARY | TEMP } EXTERNAL TABLE`

Will change the order, but is `TEMP` really needed? seems MySQL only has `temporary`